dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/Teradata14LogsConnector.java

```diff
@@ -111,12 +111,12 @@ String getSql(@Nonnull Predicate<? super String> predicate) {
     private static class LogQueryFactory extends TeradataLogsConnector.TeradataLogsJdbcTask {
 
         public LogQueryFactory(String targetPath, SharedState state, String logTable, String queryTable, String condition, ZonedInterval interval) {
-            super(targetPath, state, logTable, queryTable, condition, interval);
+            super(targetPath, state, logTable, queryTable, condition, interval, false);
         }
 
         @Override
         @Nonnull
-        String getSql(@Nonnull Predicate<? super String> predicate) {
+        String getSql(@Nonnull Predicate<? super String> predicate, String[] expressions) {
```

Comment:
I think you only have to add this, because you added the parameters to the TeradataLogsConnector.
If you instead created a `TeradataAssessmentLogsJdbcTask`  - ie. add this customization as a subclass - then yo don't have to add these (unused) parameters here.

Reply:
Agree, this parameter should not present in this class.

I removed `expressions` from this method, here is no longer needed. It is overridden only in `TeradataLogsConnector` class now, could you please take another look?